run_conformance_tests.py

```diff
@@ -127,21 +135,15 @@ def subprocess_call(parameter_list):
     return subprocess.call(parameter_list)
 
 
-def configure_and_run_tests(cmake_call, build_system_call, verbose,
-                            build_only):
+def configure_and_run_tests(cmake_call, build_system_call, build_only,
+                            ctest_call):
     """
     Configures the tests with cmake to produce a ninja.build file.
     Runs the generated ninja file.
     Runs ctest, overwriting any cached results.
     """
 
     build_system_call = build_system_call.split()
-    ctest_call = [
-        'ctest', '.', '-T', 'Test', '--no-compress-output',
-        '--test-output-size-passed', '0', '--test-output-size-failed', '0'
-    ]
-    if verbose:
-        ctest_call.append('-V')
```

Comment:
I see. You have generalized the verbose case.
I was thinking "verbose" for the Python script first...

Reply:
I guess the option was a bit deceiving before, as it referred only to CTest.